packages/victory-core/src/types/prop-types.ts

```diff
@@ -62,37 +88,21 @@ export interface EventPropTypeInterface<TTarget, TEventKey> {
   };
 }
 
-export type DomainTuple = [number, number] | [Date, Date];
-export type DomainPropObjectType =
-  | { x?: DomainTuple; y: DomainTuple }
-  | { x: DomainTuple; y?: DomainTuple };
-export type DomainPropType = DomainPropObjectType | DomainTuple;
-
-export type PaddingType = number | [number, number];
-export type DomainPaddingPropType =
-  | PaddingType
-  | {
-      x?: PaddingType;
-      y?: PaddingType;
-    };
-
-export type RangeTuple = [number, number];
-export type RangePropType = RangeTuple | { x?: RangeTuple; y?: RangeTuple };
-
-type NumberValue = number | { valueOf(): number };
+// type NumberValue = number | { valueOf(): number };
 /**
  * D3 scale function shape. Don't want to introduce typing dependency to d3
  */
-export interface D3Scale<TRange = any> {
-  (input: NumberValue): number;
-
-  ticks: (count?: number) => number[];
-  tickFormat: (count?: number) => (d: number) => string;
-  domain: () => number[];
-  range: () => TRange[];
-  copy: () => this;
-  invert: (value: number) => number;
-}
+// export interface D3Scale<TRange = any> {
+//   (input: NumberValue): number;
+
+//   ticks: (count?: number) => number[];
+//   tickFormat: (count?: number) => (d: number) => string;
+//   domain: () => number[];
+//   range: () => TRange[];
+//   copy: () => this;
+//   invert: (value: number) => number;
+// }
+export type D3Scale = any;
```

Comment:
I will keep looking into this but I have spent many hours trying to type this in a way that works for d3 scale types, and I haven't figured it out yet.